core/src/themes/ionic.functions.sizes.scss

```
@@ -0,0 +1,13 @@
+@use "sass:math";
+
+$browser-context: 16;
+
+/**
+ * Convert a pixels given value into rem
+ * 
+ * @param pixels - Value in pixels to be converted (i.e. px)
+ * @param context (optional) - Baseline value
+ */
+@function pxToRem($pixels, $context: $browser-context) {
```

```suggestion
@function px-to-rem($pixels, $context: $browser-context) {
```

Could we change this case to match the functions in other files:
- https://github.com/ionic-team/ionic-framework/blob/main/core/src/themes/ionic.functions.color.scss#L57
- https://github.com/ionic-team/ionic-framework/blob/main/core/src/themes/ionic.functions.string.scss#L7

---

Done!